### PR TITLE
Updated Openfin typings to v17

### PR DESCRIPTION
**Pull request by chrisbarker**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://developer.openfin.co/versions/?product=Runtime%20Versions&version=6.49.17.14
- [x] Increase the version number in the header if appropriate.

### CODE REVIEW

File: openfin/index.d.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ declare namespace fin {
 		 */
 		close(force?: boolean, callback?: () => void, errorCallback?: (reason: string) => void): void;
 		/**
-		 * Retrieves an array of wrapped fin.desktop.Windows for each of the application�s child windows.
+		 * Retrieves an array of wrapped fin.desktop.Windows for each of the application�s child windows.
```
Comment (minestarks) [inline openfin/index.d.ts]: Could you change these to regular apostrophes or just revert the line please?

A few more occurrences below

**Review by minestarks — CHANGES_REQUESTED** [review]

**Review by minestarks — APPROVED** [review]

**Comment by minestarks** [comment]
Now I see somehow there are merge conflicts... once you resolve them I can merge. Thank you.

**Comment by paulvanbrenk** [comment]
Thanks for the contribution. Manually merging this in.

**Comment by joneit** [comment]
@chrisbarker FYI we are currently in the process of preparing an update.
Feel free to reach out. I am at: jonathan@openfin.co

**Pull request closed without merging**